src/analysis/utils/cost-utils.ts

```diff
@@ -80,9 +79,9 @@ export const getGpuCost = (gpuType: string, numGpus: number, region: string): nu
 // This function deals with runtimes that are paused
 // All disks referenced in this function are boot disks (aka the disk google needs to provision by default for OS storage)
 // The user pd cost for a runtime is calculated elsewhere
-export const runtimeConfigBaseCost = (config: GoogleRuntimeConfig): number => {
+export const runtimeConfigBaseCost = (config: RuntimeConfig): number => {
```

Comment:
I think this can stay a GoogleRuntimeConfig. The scope of this method is GCP-only, right?